java-shared-dependencies/dependency-analyzer/src/main/java/com/google/cloud/DependencyAnalyzer.java

```diff
@@ -109,7 +112,8 @@ public static void main(String[] args) throws IllegalArgumentException {
       System.exit(1);
     }
 
-    ReportResult result = analyzeReport.generateReport();
+    System.out.println(analyzeReport.toString());
```

Comment:
Insert `System.out.println("Please copy and paste the package information below to your ticket.")`.

The string representation of analyzeReport should not contain the message.

Reply:
Done.